### PR TITLE
Fix errors where OCI hooks directory does not exist

### PR DESCRIPTION
**Pull request by mheon**
Hopefully fixes https://bugzilla.redhat.com/show_bug.cgi?id=1657263

### CODE REVIEW

File: libpod/container_easyjson.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// +build  seccomp   ostree selinux  varlink exclude_graphdriver_devicemapper
+// +build  seccomp   containers_image_ostree_stub selinux  varlink exclude_graphdriver_devicemapper
```
Comment (rhatdan) [inline libpod/container_easyjson.go]: Is this related.

Comment (mheon) [reply libpod/container_easyjson.go]: Oops - looks like easyjson regenerated. I'll fix.

**Comment by rhatdan** [comment]
Tested it locally and it works.
/lgtm

**Comment by cevich** [comment]
Since (presumably) Dan found the bug in Fedora, might be nice to rebase so the F29 cirrus test can chew on it.

**Comment by cevich** [comment]
gaa! I was 12-seconds too slow :(

**Pull request merged**